e3db/client.py

```diff
@@ -41,6 +42,8 @@ def __init__(self, config):
         self.client_id = config['client_id']
         self.public_key = config['public_key']
         self.private_key = config['private_key']
+        self.public_signing_key = config['public_signing_key']
+        self.private_signing_key = config['private_signing_key']
         self.e3db_auth = E3DBAuth(self.api_key_id, self.api_secret, self.api_url)
```

Comment:
Let's instantiate a tsv1_auth here. Then we have it for the next set of work

Reply:
Added it alongside e3db_auth